cpu/sam0_common/Makefile.include

```diff
@@ -28,7 +28,7 @@ CFLAGS += -DDONT_USE_PREDEFINED_PERIPHERALS_HANDLERS
 # For Cortex-M cpu we use the common cortexm.ld linker script
 LINKER_SCRIPT ?= cortexm.ld
 
-# define sam0 specific pseudomodules
-PSEUDOMODULES += sam0_periph_uart_hw_fc
+# include sam0 common periph drivers
+USEMODULE += sam0_common_periph
```

Comment:
unrelated

Reply:
I screw up during the last rebase sorry...
Fixed !